modin/pandas/test/dataframe/test_indexing.py

```diff
@@ -1699,6 +1699,7 @@ def test_tail(data, n):
 
 
 def test_xs():
+    # example is based on the doctest in the upstream pandas docstring
```

Comment:
We should add more test cases (including distributed cases, i.e. number of partition is other than 1) to verify the new implementation.

Reply:
i updated the test to have multiple partitions

Reply:
@YarShev done, ready for review

Reply:
There are many lines not covered by codecov. Should we expand tests?

Reply:
Perhaps there was some mistake with `codecov`, let's first see how it will be this time.